wiki/People/The_Team/Beatmap_Nominators/Expectations/en.md

```diff
@@ -0,0 +1,17 @@
+# Expectations for Beatmap Nominators
+
+*See also: [Beatmap Nominator Rules](/wiki/People/The_Team/Beatmap_Nominators/Rules)*
+
+As central figures in their respective communities and osu! as a whole, [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) are held to a higher standard than other users. The expectations listed here are the result of discussion within the [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team). While it's unreasonable to perfectly align with these expectations, going far out of line can lead to warnings and/or removal from the Beatmap Nominators. All situations are judged on a case-by-case basis depending on frequency, severity, judgement, and impact.
```

Comment:
> judged on a case-by-case basis depending on frequency, severity, judgement, and impact.

not sure what "judgement" is referring to here and it doesn't seem necessary

Reply:
hm, might not be the best word then
wanted a summarizing way of referring to the difference between a case being obvious (applying common sense) and a case where someone encounters a more difficult situation and makes a poor, but understandable, decision
having written that out, "obviousness" or even "common sense" could be a suitable replacement then?